lib/keisan/ast/plus.rb

```diff
@@ -39,25 +39,25 @@ def simplify(context = nil)
         # Commutative, so pull in operands of any `Plus` operators
         @children = children.inject([]) do |new_children, cur_child|
           case cur_child
-          when AST::Plus
+          when Plus
             new_children + cur_child.children
           else
             new_children << cur_child
           end
         end
 
-        if children.all? {|child| child.is_a?(Keisan::AST::String)}
-          return Keisan::AST::String.new(children.inject("") do |result, child|
+        if children.all? {|child| child.is_a?(String)}
+          return String.new(children.inject("") do |result, child|
```

Comment:
I should have realized that namespacing rules means that `String` here will identify with `Keisan::AST::String` before the built in `String`.  Nice simplification here.